tutorials/rendering/renderers.rst

```diff
@@ -0,0 +1,315 @@
++---------------------+--------------------------+--------------------------+--------------------------+
+| Web                 | ✔️ Yes.                  | ❌ No.                   | ❌ No.                   |
++---------------------+--------------------------+--------------------------+--------------------------+
+| 2D Games            | ✔️ Yes.                  | ✔️ Yes, but              | ✔️ Yes, but              |
+|                     |                          | Compatibility is usually | Compatibility is usually |
+|                     |                          | good enough for 2D.      | good enough for 2D.      |
++---------------------+--------------------------+--------------------------+--------------------------+
+| 3D Games            | ✔️ Yes.                  | ✔️ Yes.                  | ✔️ Yes.                  |
++---------------------+--------------------------+--------------------------+--------------------------+
+| **Feature set**     | 2D and core 3D features. | Most rendering features. | All rendering features.  |
++---------------------+--------------------------+--------------------------+--------------------------+
+| 2D rendering        | ✔️ Yes.                  | ✔️ Yes.                  | ✔️ Yes.                  |
+| features            |                          |                          |                          |
++---------------------+--------------------------+--------------------------+--------------------------+
+| Core 3D rendering   | ✔️ Yes.                  | ✔️ Yes.                  | ✔️ Yes.                  |
+| features            |                          |                          |                          |
++---------------------+--------------------------+--------------------------+--------------------------+
+| Advanced            | ❌ No.                   | ✔️ Yes, limited by       | ✔️ Yes. All rendering    |
+| rendering features  |                          | mobile hardware.         | features are supported.  |
++---------------------+--------------------------+--------------------------+--------------------------+
+| New features        | ⚠️ Some new rendering    | ✔️ Most new rendering    | ✔️ All new features are  |
+|                     | features are added to    | features are added to    | added to Forward+. As the|
+|                     | Compatibility. Features  | Mobile. Mobile usually   | focus of new development,|
+|                     | are added after Mobile   | gets new features as     | Forward+ gets features   |
+|                     | and Forward+.            | Forward+ does.           | first.                   |
++---------------------+--------------------------+--------------------------+--------------------------+
+| Rendering cost      | Low base cost, but       | Medium base cost, and    | Highest base cost, and   |
+|                     | high scaling cost.       | low scaling cost.        | low scaling cost.        |
```

Comment:
Mobile does not use clustered lighting, so it doesn't scale as well to complex scenes as Forward+:

```suggestion
| Rendering cost      | Low base cost, but       | Medium base cost, and    | Highest base cost, and   |
|                     | high scaling cost.       | medium scaling cost.        | low scaling cost.        |
```